gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/user/model/BaseUserEntity.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.apim.core.user.model;
+
+import java.util.Date;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class BaseUserEntity {
+
+    /** User identifier */
+    private String id;
+    /** The organization id of the user */
+    private String organizationId;
+    /** The user first name */
+    private String firstname;
+    /** The user last name */
+    private String lastname;
+    /** The user email */
+    private String email;
+    /** The user creation date */
+    private Date createdAt;
+    /** The user last updated date */
+    private Date updatedAt;
+    /** The source when user is coming from an external system (LDAP, ...) */
+    private String source;
+    /** The user reference in the external source */
+    private String sourceId;
```

Comment:
do we need the comments ?

Reply:
we can get rid of some of them. Maybe we can keep `source` and `sourceId` 🤔